<launch>
	<!--Load config file for rosconsole so that log level for scene recognition is set to debug.-->
	<env name="ROSCONSOLE_CONFIG_FILE" value="$(find asr_recognizer_prediction_psm)/launch/rosconsole_asr.config"/>
        <node name="asr_recognizer_prediction_psm" pkg="asr_recognizer_prediction_psm" type="asr_recognizer_prediction_psm" output="screen">
        
	  <!-- print additional messages to the console -->
	  <param name="print_debug_messages" value="false"/>
	  
	  <!-- generate a RViz MarkerArray and publish it -->
	  <param name="visualize_hypothesis" value="true"/>
	  <param name="marker_lifetime" value="20.0"/>
	  
	  
	  <!-- deprecated. The service call now returns the hypothesis. -->
	  <param name="publish_hypothesis" value="false"/>
	  <param name="publish_hypothesis_topic" value="pose_prediction_psm_hypothesis"/>
        </node>
</launch>
